src/arrayMethodSort.js

```diff
@@ -3,9 +3,38 @@
 /**
  * Implement method Sort
  */
+
+function initialSort(a, b) {
+  if (`${a}` > `${b}`) {
```

Comment:
it's better to use the String() constructor, and add two variables to store strings values, to not duplicate the logic and not call the constructor every time